CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ I am committed to that goal and more than happy to help to you learn to code and
 ```
 git clone https://github.com/bthayer2365/git-gud
 cd git-gud
-pip install -e . # Installs in edit mode so we can change files without needing to reinstall
+pip install -e . # NB: Do not overlook the `.` after the -e option! Installs in edit mode so we can change files without needing to reinstall

```

Comment:
Great comment. I can see how someone could miss that. Can you add this to README.md too?